tasks/AccelerationController.cpp

```diff
@@ -31,10 +31,42 @@ bool AccelerationController::configureHook()
     cmdVector = Eigen::VectorXd::Zero(numberOfThrusters);
     expectedEffortVector = Eigen::VectorXd::Zero(6);
     names = _names.get();
+    base::MatrixXd weighingMatrix;
+    base::VectorXd thrustersWeights = _thrusters_weights.get();
+
+
+	if(thrustersWeights.size() != numberOfThrusters)
```

Comment:
You're using tabs, and have a tabsize of 4. Indentation is broken as a result. Only use spaces and a shift size of 4 to indent (check out your editor's configuration, you should be able to tell it to only use spaces)